src/web/wms/src/main/java/applicationContext.xml

```diff
@@ -59,5 +59,6 @@
      <property name="icon" value="palette.png"/>
      <property name="order" value="30"/>
      <property name="authorizer" ref="workspaceAdminAuthorizer"/>
-  </bean>  
+  </bean>
```

Comment:
?

Reply:
that's eclipse doing its thing, I will try to revert it

Reply:
kk